src/libp2p_config.erl

```diff
@@ -119,20 +120,15 @@ remove_pid(TID, Kind, Ref) ->
 
 -spec remove_pid(ets:tab(), pid()) -> true.
 remove_pid(TID, Pid) ->
-    ets:insert(TID, {{delete, Pid}, Pid}),
-    %spawn(fun() ->
-    %              ets:match_delete(TID, {'_', Pid}),
-    %              ets:match_delete(TID, {{?ADDR_INFO, '_', Pid}, '_'}),
-    %              ok
-    %      end),
+    ets:insert(TID, {{?DELETE, Pid}, Pid}),
```

Comment:
Shouldn't we let the ets call return `true` here and make the spec allow for a badarg result?